fs/fs_zenfs.cc

```diff
@@ -272,6 +272,11 @@ IOStatus ZenFS::Repair() {
   return IOStatus::OK();
 }
 
+std::string ZenFS::FormatPathLexically(std::string filepath) {
+  std::filesystem::path f = filepath;
+  return f.lexically_normal().string();
+}
+
```

Comment:
We can take a `std::filesystem::path` as argument here, conversion from `std::string` is implicit. If we want to keep `std::string`, take the string as `const&` to avoid invoking copy constructor on function invocation.

Reply:
Ok